### ISSUE TITLE
Looking for Contributors

### ISSUE THREAD

**Issue by RONNCC** [pytube/pytube#698]
Looking for contributors to help pytube continue to be the leader in the Python-Youtube space 😄 

Greatly would appreciate contributions in the following areas
- Feature Development
- Testing
- CI Integration
- Docs
- StackOverflow Answers
and lots more 🎉 

Feel free to ping me if this sounds interesting to you!

**Comment by RONNCC** [comment]
@H4KKR

**Comment by belthaZornv** [comment]
If you need anything, would be happy to help (dev)

**Comment by EmptySpace99** [comment]
@RONNCC I can help you to improve this project regarding the CI integration area 😃

**Comment by RONNCC** [comment]
@belthaZornv @EmptySpace99 

(more immediately) if you have some time would love assistance with cleaning up the giant PR/Issue backlog 😓  - having difficulty telling what is still relevant / needs to be fixed

Feel free to @ me on anything I can close

**Comment by H4KKR** [comment]
Hey @RONNCC I'd be more than happy to help!! I'm happy to merge and shut down pytubeX, it was just an alternative to pytube3 who's bugs drove me insane!! On further inspection, I don't think pytube uses the same cypher.py logic so isn't affected by the same bug. Let me know if it's any different and I'll take a look.
I would absolutely love to help on this project, especially if we could work it up to be high-profile! Let me know if there's anything I can do to help. I'm also really eager to help with issue backlogging and PR, if you let me know what to do I'll do it!!

Thanks guys!

**Comment by pgirikishore** [comment]
Hey @RONNCC if there's anything you need, I'll be more than happy to help.

**Comment by jayathungek** [comment]
I've been using pytube as part of a little project and I think it's a great tool. Would be glad to help out maintaining it/fixing bugs if needed!

**Comment by hoffmaps** [comment]
I'm not too experienced with everything, but I'd love to learn more and contribute to this project! (Coming from Internhacks 2020)
I do have python experience and think it would be cool to do something more with that :)

**Comment by HannahGuo** [comment]
I have some experience with Python, and I'd also love to learn more and contribute! I came from your Internhacks 2020 post :)

**Comment by Ksengine** [comment]
how can i contribte

**Comment by conormora** [comment]
@RONNCC Hi, interested in contributing. Please let me know if any tasks are still open.

**Comment by David-Billingsley** [comment]
@RONNCC I am interested in helping, I am still learning python and will be a little slower.  Please let me know if their are any open tasks.

**Comment by ghostxzxzgit** [comment]
@RONNCC would love to contribute and upkeep.

**Comment by a-czyrny** [comment]
The last real code commit was on 30.07.2020. I have already forked the repository and implemented the latest pull request myself. In addition also missing attributes like the publish date. What do the future of this repository look like?

**Comment by RONNCC** [comment]
@a-czyrny need to make a mailing list and get a integration test suite up. 

The latter is the only real blocker to PRs here 😄

**Comment by RONNCC** [comment]
@H4KKR  @belthaZornv @EmptySpace99 @pgirikishore @jayathungek  @hoffmaps  @HannahGuo  @Ksengine @conormora @David-Billingsley  @ghostxzxzgit  @a-czyrny  

Can you join the mailing list -- would speed up convos and we can assign issues / etc

https://groups.google.com/g/pytube-dev/about

**Comment by tfdahlin** [comment]
hey @RONNCC I sent a request to join that mailing list

**Comment by haelmj** [comment]
Hey, @RONNCC I have some experience using python. I'd love to contribute as well. I'll send a request to join the list.

**Comment by RONNCC** [comment]
Michael sounds good - I look forward to it

**Comment by DeepakChakravarthy** [comment]
@RONNCC I Would Like to Contribute and Enhance Pytube 
May I Join?

**Comment by tfdahlin** [comment]
@DeepakChakravarthy absolutely, Ronnie and I try to keep up with the issues and PRs that come in, so if you have an idea for something that could improve pytube, feel free to create either! You can also request to join the mailing list, and one of us will approve it at some point when we're able to

**Comment by RamkumarMrj** [comment]
@RONNCC @tfdahlin I have an idea to build-up the pytube there by I would like to contribute.

**Comment by RONNCC** [comment]
@RamkumarMrj  awesome! Could you 

- join the mailing list (let's you message us)
- create an issue with the feature idea?

This lets us discuss ideas in an open forum where everyone can see the rationale for design decisions later on!

**Comment by sneha-001** [comment]
@RONNCC Hi Ronnie! I would like to contribute to pytube! I'll make sure to join the mailing list.

**Comment by Pacifist2503** [comment]
Hello there! Is it possible to use pytube to download videos from youtube through your account?

**Comment by tfdahlin** [comment]
@Pacifist2503 no, pytube does not support logging in.

**Comment by mahmoud-abdallah863** [comment]
@RONNCC I'm interested,
I can't do development and testing stuff. But I'm cool with other things

**Comment by yvainyork** [comment]
@RONNCC : from time to time I could also help (develop ideas, testing, documentation)
Very nice project :)

**Comment by RONNCC** [comment]
cc @tfdahlin fyi at above

@yvainyork  @mahmoud-abdallah863  would love your assistance -- can you join the mailing DL https://groups.google.com/g/pytube-dev

**Comment by yrzaa** [comment]
@RONNCC Would love to contribute! Shall I join the mailing list too? I like how lightweight this is and have some ideas for it. I'm a bit of a beginner, but I'm learning something new every day

**Comment by shivanishimpi** [comment]
Would be gladly interested to contribute, I've posted one of the enhancements that would be very helpful for the community. Have requested to join the mailing list, please do let me know if there's something else that needs to be done.

**Comment by LukasKaspras** [comment]
Where could I join the mailing list? Would like to contribute as well!

**Comment by jcs090218** [comment]
I would recommend a discord server. :)

**Comment by MASHOD0** [comment]
I am new to open source, I want to help with anything I can . (I am kinda new to python also )

**Comment by 0xbrayo** [comment]
@RONNCC I'm interested

**Comment by JarbasAl** [comment]
i just started using this lib instead of maintaining a similar one, i will be sending PRs here and there starting today 

I am not familiar with the code base yet and don't have much time available, but i am happy to lend a hand

**Comment by Krystelarekt** [comment]
I am using this library for a project of my own and i would be happy to help out fixing bugs (if you are still interested in making me a contributor). I dont understand the code very well right now so if anyone here can help me out that would be great too.

**Comment by jsharmashell** [comment]
Pls know me if can try any stuff . Similar to others I have started using this Pytube since last December :) . Happy to contribute if I can make sense.
Stackoverflow - I usually answers about using pytube . But yes i am not having very deep knowledge on this repo. Feel we can learn together.

**Comment by Krystelarekt** [comment]
@jsharmashell yes, why not? I am currently working on a mass song downloader (more than 1000 songs) using pytube. Will upload to github soon. Feel free to use my code any way you want.

**Comment by ExcitedRutvik** [comment]
@RONNCC  I'm new to this but would love to learn and try contributing.

**Comment by milkiyd** [comment]
Hello @RONNCC , I'm new to programming and open source projects and I would like to contribute to this project

**Comment by glubsy** [comment]
Don't just spam this thread with "I'm willing to help" messages. We don't care.

Either contribute directly or refrain from spamming. This is an open-source project, you have full access to the source code already. So get hacking, submit your patches as merge requests, and don't waste anybody's precious time.

**Comment by rexwu1104** [comment]
@RONNCC I pull a pull request for API Call, I think it can help.
I just a student, so, I can't use computer more time. I wish it can help you.

**Comment by wkrettek** [comment]
I would like to contribute to the repo. Are any maintainers watching this thread? I tried to join the mailing list as well

@RONNCC would love if I could get in touch with you and ask about the current state of the repo

**Comment by poneoneo** [comment]
Hey, @RONNCC I really like to work with  python. I'm working on a personal project to build a website to download youtube playlist video so i really want to contribute to pytube  .how can i join a mailling list ???

**Comment by FahimFBA** [comment]
Hello,

I was thinking about creating a GUI application for this project.

Please let me know what do you think about this. Initially it can be simple with tkinter with all the textboxes, guidelines in it.

**Comment by abhradeepde123** [comment]
@RONNCC i'd love to implement some features

**Comment by davidegazze** [comment]
Can I help you to implement the features

**Comment by Corny69** [comment]
Can I contribute?

**Comment by matrusto** [comment]
I have time to dedicate to this and started to offer fixes.

**Comment by samirul** [comment]
Will be happy to help  😊

**Comment by v-danh** [comment]
> Hey @RONNCC I'd be more than happy to help!! I'm happy to merge and shut down pytubeX, it was just an alternative to pytube3 who's bugs drove me insane!! On further inspection, I don't think pytube uses the same cypher.py logic so isn't affected by the same bug. Let me know if it's any different and I'll take a look. I would absolutely love to help on this project, especially if we could work it up to be high-profile! Let me know if there's anything I can do to help. I'm also really eager to help with issue backlogging and PR, if you let me know what to do I'll do it!!
> 
> Thanks guys!

@H4KKR Can you solve the problem in #2051? Many thanks for considering my request.

**Comment by olegcherev** [comment]
@RONNCC what do you think of implementing the gamification elements (XP/Levels/Badges/etc) to encourage the contributors?

**Comment by pam-param-pam** [comment]
What is the state of this Library? There was not a single commit for more than a year, there are hundreds of open issues, many of which are major, breaking bugs that make this library absolete. @RONNCC, can we get some information? Is this project deprecated? 

And to everyone saying they would love to contribute. Please do so, fork and make merge requests!

**Comment by TechPsychotic** [comment]
Hi, I have been using pytube from 7 years and really enjoyed it and would like to contribute to it. Although I am currently facing some issues in pytube like many others, I would really like to be part of the team that fixes or tests it for later versions. @RONNCC
